common/thorhelper/thorxmlwrite.cpp

```diff
@@ -149,3 +150,147 @@ extern thorhelper_decl void convertRowToJSON(size32_t & lenResult, char * & resu
     rtlStrToStrX(sizeResult, result, writer.length(), writer.str());
     lenResult = rtlUtf8Length(sizeResult, result);
 }
+
+
+//=============================================================================================
+
+void PropertyTreeXmlWriter::outputLiteralString(size32_t size, const char *value, const char *fieldname)
+{
+    //Horrible that there is no interface for adding a (len, data) string
+    StringBuffer temp;
+    temp.append(size, value);
+    root->addProp(fieldname, temp.str());
+}
+
+
+void PropertyTreeXmlWriter::outputInlineXml(const char *text)
+{
+    UNIMPLEMENTED;
```

Comment:
Are these going to be implemented later (in which case is there a Jira), or should they be "throwUnexpected()" ?

Reply:
changed